### PR TITLE
Added: Navigation

### PR DESCRIPTION
**Pull request by sushil-mod**
Added navigation
nav

### CODE REVIEW

File: Components/nav.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    padding-right: 2rem;
+}
+.nav-login-btn{
+font-size: 1.2rem;
+padding: 0.5rem;
+border: 2px solid var(--text-color);
+color: var(--text-color);
+background-color: var(--white);
+border-radius: 10px;
+}
```
Comment (abhijitdotsharma) [inline Components/nav.css:44]: Use a code formatter, prettier/beautify extension on VS Code

**Review by abhijitdotsharma — APPROVED** [review]
Rest looks good

**Review by atharvabhanage02 — APPROVED** [review]
Just one suggestion : You can remove the blank lines at the end of the HTML file. 
Else everything looks fine.

File: Components/nav.css (hunk 1 above)
Comment (sushil-mod) [reply Components/nav.css:44]: thanks will do now

**Pull request merged**